BurnerB/PropertyProLite-CH2
# Ch cloudinary config 167164224

#### What does this PR do?
Adds cloudinary cloud image hosting to api
#### Description of Task to be completed?
A users image file should be stored as an image link,a link that leads to the saved image on cloudinary.
#### How should this be manually tested?
clone this repo
run `npm start`
visit the link on postman:
POST `http://127.0.0.1:5000/api/v1/property`
with valid input data
```
          price: 5000000,
          state: 'Nairobi',
          city: 'Nairobi',
          address: 'Kenya',
          type: '2 bedroom',
          image_url: 'change-wordpress-url-1.png'
```

You should recieve a response with an image url

```
 "status": "success",
    "data": {
        "_id": 1,
        "owner": 2,
        "ownerEmail": "johndoe@gmail.com",
        "ownerPhoneNumber": "0727988632",
        "status": "Available",
        "type": "1 bedroom",
        "state": "Kenya",
        "city": "Nairobi",
        "price": "100000",
        "address": "Kenya",
        "created_on": "July 10th 2019, 7:44:17 am",
        "image_url": "http://res.cloudinary.com/burnerb/image/upload/v1562733865/atqe57h6a8gnvn70c64q.png"
```
#### What are the relevant pivotal tracker stories?
[#167153682](https://www.pivotaltracker.com/story/show/167164224)
#### Screenshots (if appropriate)

![Cloudinary](https://user-images.githubusercontent.com/45785786/60962032-0c26fb00-a316-11e9-9d8b-3161c3185e44.png)
